transformer_uproot.py

```diff
@@ -108,6 +114,11 @@ def callback(channel, method, properties, body):
         channel.basic_publish(exchange='transformation_failures',
                               routing_key=_request_id + '_errors',
                               body=json.dumps(transform_request))
+
+        servicex.post_status_update(file_id=_file_id,
+                                    status_code="failure",
+                                    info="error: "+str(exc_value)[0:1024])
```

Comment:
This isn't enough info - you'll want the dump of the traceback as well, I think.

Reply:
The idea is that there is still plenty of info in the log file as well as in the rabbitMQ dead letter queue. This is going into a database and I just wanted to make sure there was enough info to find lost files and to clue me in on which pod's log to examine. 
If we wanted to extend this to full details then we might want elastic search or switch the app db to use full text blobs

Reply:
are transformer logs saved?

Reply:
In the Postgres DB - you can access them with a port forward of 5432